source/developer/slash-commands.rst

```diff
@@ -3,6 +3,46 @@
 Slash Commands
 ==============
 
+Intro 
+
+.. toctree::
+	:maxdepth: 2
+
+Creating a simple slash command
+-----------------------------------
+
+Parameters and formatting
+--------------------------
+
+Subheading
```

Comment:
The slash commands and outgoing webhooks docs are WIP.

Just the incoming webhooks is ready for review for content and structure